Source/Game/Game/Util/ImGui/FakeScrollingArea.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+#include <imgui/imgui.h>
+
+#include <vector>
```

Comment:
These included headers aren't needed here, please try to avoid including unnecessary stuff since it increases compilation times. We're not great at enforcing this ourselves, but <functional> is one of the worst offenders. :P